src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -208,15 +207,21 @@ def wf_logs(
 @workflow.command()
 @cloup.argument("wf_run_id", required=False)
 @CONFIG_OPTION
-def stop(wf_run_id: t.Optional[str], config: t.Optional[str]):
+@cloup.option(
+    "--force/--no-force",
+    is_flag=True,
+    default=None,
+    help="Will forcefully terminate a workflow, "
+    "without waiting for it to gracefully exit.",
```

Comment:
**suggestion**: could you describe what happens when neither `--force` nor `--no-force` is passed?

Reply:
Wdyt 5537ad3?